File: pinyougou-interface/src/main/java/cn/itcast/core/service/order/OrderService.java
package cn.itcast.core.service.order;

import cn.itcast.core.entity.PageResult;
import cn.itcast.core.pojo.order.Order;

import java.util.Date;

public interface OrderService {

    /**
     * 保存订单
     * @param username
     * @param order
     */
    public void add(String username, Order order);

    /**
     * 待审核的商家的统计
     * @param page
     * @param rows
     * @param startDate
     * @param endDate
     * @return
     */
    public PageResult statistics(Integer page, Integer rows, Date startDate,Date endDate,Order order);

    /**
     * 运营商系统订单查询
     * @param page
     * @param rows
     * @param minPrice
     * @param maxPrice
     * @param order
     * @return
     */
    PageResult search(Integer page, Integer rows, String minPrice, String maxPrice, Order order);

}

File: pinyougou-web-manager/src/main/webapp/js/service/orderService.js
//服务层
app.service('orderService',function($http){
	    	
	//读取列表数据绑定到表单中
	this.findAll=function(){
		return $http.get('../order/findAll.do');
	}
	//分页 
	this.findPage=function(page,rows){
		return $http.get('../order/findPage.do?page='+page+'&rows='+rows);
	}
	//查询实体
	this.findOne=function(id){
		return $http.get('../order/findOne.do?id='+id);
	}

	//统计
	this.search=function(page,rows,createTime,searchEntity){
		return $http.post('../order/statistics.do?page='+page+"&rows="+rows+"&createTime="+createTime, searchEntity);
	}

	//查询
    this.search=function(page,rows,minPrice,maxPrice,searchEntity){
        return $http.post('../order/search.do?page='+page+"&rows="+rows+"&minPrice="+minPrice+"&maxPrice="+maxPrice, searchEntity);
    }

});

